hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -360,10 +367,12 @@ private void handleOperationError(Operation operation, Throwable e) {
 
     private void sendResponseAfterOperationError(Operation operation, Throwable e) {
         try {
-            if (node.getState() != NodeState.SHUT_DOWN) {
+            if (node.getState() == NodeState.SHUT_DOWN) {
```

Comment:
Do not see a case for this in the offloaded path.

Reply:
The behavior remains unchanged compared to the master behavior. 

PS: I have reverted this code cleanup because it could cause confusion and want to keep the change simple. 

But I'll add it to the list of todo's.